lib/graphql/schema.ts

```diff
@@ -161,6 +163,11 @@ export type UserInvoiceArgs = {
 };
 
 
+export type UserInvoicingDashboardDataArgs = {
+  pageNumber: Scalars['Float'];
```

Comment:
Should it rather be `INTEGER` ? I mean a page will always be a whole number, there will never be decimal digits.

Reply:
In BE its declared like this:
`@Arg("pageNumber") pageNumber: number`

If you search for 'float' in the repo, there are multiple places where its the same, so I think its fine 🤔

Reply:
Agree. I think it is fine too.